File: Python/paypal.py
# Importamos las clases
from payment import Payment

# Clase Paypal (Hereda de la clase Pago)
class Paypal(Payment):

    # Atributos o propiedades de la clase
    id = int
    email = str
    
    # Método constructor
    def __init__(self, email):
        super().__init__()
        self.email = email

File: Java/Paypal.java
// Clase Paypal (Hereda de la clase pago)
class Paypal extends Payment {
    // Atributos o propiedades de la clase
    Integer id;
    String email;

    // Método constructor
    public Paypal(String email) {
        super();
        this.email = email;
    }
}